frontend/src/pages/ContactUs/ContactUs.jsx

```diff
@@ -115,6 +140,6 @@ export const ContactUs = (props) => {
           </div>
         </div>
       </div>
-    </div>
```

Comment:
Hey @AbhipsaGuru1012 while resolving the conflict, you left some bug(probably deleted closing div and braces from here) Please fix it. (You would be able to figure out the error just by runing this branch's code)

Reply:
@Kajol-Kumari I have made some modifications. Please let me know if this is okay.